user-service/src/main/java/com/microcommerce/userservice/data/entity/UserRole.java

```diff
@@ -10,10 +10,12 @@
 @Getter
 @Setter
 @NoArgsConstructor
+@Builder
+@AllArgsConstructor
 public class UserRole {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.UUID)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
```

Comment:
Why sequence for this table?

Reply:
Type of the id here is "Long" so the strategy can't be GenerationType.UUID, that change broke the code when it was introduced.